api/src/main/proto/bio/terra/tanagra/underlay/search.proto

```diff
@@ -0,0 +1,81 @@
+// Messages describing how to construct Tanagra entity searches.
+
+syntax = "proto3";
+
+package bio.terra.tanagra.underlay;
+option java_package = "bio.terra.tanagra.proto.underlay";
+option java_multiple_files = true;
+
+import "bio/terra/tanagra/underlay/entities.proto";
+import "bio/terra/tanagra/underlay/mapping.proto";
+
+// The schema of how an entity filter can be constructed for a given entity.
+message EntityFiltersSchema {
```

Comment:
Same question as on the other PR: are these properties of the filters, or properties of the entity attributes themselves? Is the schema sufficient to express what search operations are possible, or do we need to constraint / expand upon that with this additional structure?

Reply:
I think these are properties of the filters. For example, we might not want to suggest users filter on every attribute on the entity - e.g. we don't need selectors for both `person.gender` and `person.gender_concept_id`, we would just have an EnumHint for `person.gender_concept_id`.
We might need to further expand. Right now the supported operations on the DataType are implicit (not that we've defined many operations yet, just `EQUALS` and `LESS_THAN`). We might want some special string term search operator that's not available on every `STRING` entity, at which point we might want to enumerate the supported attributes, or add sub types for DataTypes. Not sure.

Reply:
Thanks, I think I'm convinced.